ros2_batch_job/osx_batch/__init__.py

```diff
@@ -36,16 +36,19 @@ def pre(self):
         else:
             warn('ccache does not appear to be installed; not modifying PATH')
         if 'LANG' not in os.environ:
+            warn('LANG unset; using default value')
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
         if 'OPENSSL_ROOT_DIR' not in os.environ:
+            warn('OPENSSL_ROOT_DIR not set; installing openssl')
             brew_openssl_prefix_result = subprocess.run(
                 ['brew', '--prefix', 'openssl'],
                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
             if not brew_openssl_prefix_result.stderr:
               os.environ['OPENSSL_ROOT_DIR'] = brew_openssl_prefix_result.stdout.decode().strip('\n')
```

Comment:
maybe we should warn or raise in the `else` case though (openssl env var not set AND not found using `brew --prefix`)

Reply:
I made it raise in 66980c6 because I assume this is a case that is not expected to happen in practice.

Reply:
:+1: I think the build would fail if it's not set, so better fail early